src/CSS/Banner.module.css

```diff
@@ -57,7 +57,7 @@
 
 .BannerImage {
   z-index: 10;
-  width: 700px;
+  width: 100%;
   height: 400px;
```

Comment:
Increase height to 550px
Add margin-top: -15%

And remove bottom padding from jumbotron to cut extra space above 'About Us' section.

Rest everything looks good.